### PR TITLE
Update open passage to horizontal docks for Main Caves

### PR DESCRIPTION
**Pull request by JeffGainsNGames**
Docks for Main Caves updated to Horizontal Docks.

### CODE REVIEW

File: randovania/games/am2r/logic_database/Main Caves.txt [hunk 1]
```diff
@@ -17,9 +17,9 @@ Extra - minimap_data: [{'x': 16, 'y': 3}, {'x': 16, 'y': 4}, {'x': 16, 'y': 5},
   > Event - Victory
       After Collected the Baby Metroid
 
-> Dock to Hornoad Hallway; Heals? False
+> Horizontal Dock to Hornoad Hallway; Heals? False
   * Layers: default
-  * Open Passage to Hornoad Hallway/Dock to Landing Site
+  * 3-Tiles High Dock to Hornoad Hallway/Horizontal Dock to Landing Site
```
Comment (Miepee) [inline randovania/games/am2r/logic_database/Main Caves.txt:22]: should be 4 tiles high

File: randovania/games/am2r/logic_database/Main Caves.txt [hunk 2]
```diff
@@ -1129,13 +1129,13 @@ Extra - minimap_data: [{'x': 42, 'y': 26}, {'x': 43, 'y': 26}, {'x': 44, 'y': 26
           # Hang off first platform
           Power Grip and Movement (Beginner)
 
-> Area Transition to Research Site Elevator; Heals? False
+> Horizontal Dock to Research Site Elevator; Heals? False
```
Comment (Miepee) [inline randovania/games/am2r/logic_database/Main Caves.txt:1132]: i wonder whether we should make these a new type? they're at least a unique door.

Comment (Miepee) [reply randovania/games/am2r/logic_database/Main Caves.txt:1132]: would be useful as a) more door variety b) a fully trivial door in the case we ever go back to the concept of making power beam an item.

Comment (Miepee) [reply randovania/games/am2r/logic_database/Main Caves.txt:1132]: could also just slap a todo and call it a day if we wanna have that discussion for another day

Comment (JeffGainsNGames) [reply randovania/games/am2r/logic_database/Main Caves.txt:1132]: Do we really need more doors? I'll concede to the TODO for later discussion.

**Pull request merged**